runtime/Cpp/runtime/src/support/Arrays.h

```diff
@@ -32,8 +32,13 @@ namespace antlrcpp {
         return false;
 
       for (size_t i = 0; i < a.size(); ++i) {
+        if (!a[i] && !b[i])
+          continue;
+        if (!a[i] || !b[i])
+          return false;
```

Comment:
Not an issue per se, just mentioning that these two checks can be combined into one:

```cpp
     if (!a[i] != !b[i])
        continue;
```